src/databricks/azext_databricks/custom.py

```diff
@@ -37,6 +38,7 @@ def create_databricks_workspace(cmd, client,
     _set_parameter_value(parameters, 'custom_private_subnet_name', custom_private_subnet_name)  # str
     _set_parameter_value(parameters, 'prepare_encryption', prepare_encryption)
     _set_parameter_value(parameters, 'require_infrastructure_encryption', require_infrastructure_encryption)
+    _set_parameter_value(parameters, 'enable_no_public_ip', enable_no_public_ip)

```

Comment:
Does this function support nested path like 'foo.bar'?

Reply:
Currently there is only one layer properties under `parameters` and the `_set_parameter_value` method only supports a single-value path as well. 

The swagger for this service is a bit different from others. Each property is defined as an object which has `type` and `value` properties (`type` is readonly and we only need to set `value`) instead of a simple string or boolean, that's why I defined this method to set value.